WordPressCom-Stats-iOS/WPStatsService.m

```diff
@@ -179,6 +180,7 @@ - (void)retrieveInsightsStatsWithAllTimeStatsCompletionHandler:(StatsAllTimeComp
     id allTimeData = cacheDictionary[@(StatsSectionInsightsAllTime)];
     id insightsData = cacheDictionary[@(StatsSectionInsightsMostPopular)];
     id todayData = cacheDictionary[@(StatsSectionInsightsTodaysStats)];
+    id latestPostData = cacheDictionary[@(StatsSectionInsightsLatestPostSummary)];
     id commentsAuthorData = cacheDictionary[@(StatsSubSectionCommentsByAuthor)];
     id commentsPostsData = cacheDictionary[@(StatsSubSectionCommentsByPosts)];
     id tagsCategoriesData = cacheDictionary[@(StatsSectionTagsCategories)];
```

Comment:
In this case I am bastardizing the dictionary by relying upon mixed data coming back. Dangerous, I know!!